create_spark_session.py

```diff
@@ -75,15 +75,15 @@ def create_spark_session(request: Any) -> SparkSession:
         os.environ["SPARK_HOME"] = "/usr/local/opt/spark"
     clean_spark_dir()
     master = "local[2]"
-    jars = [
-        "mysql:mysql-connector-java:8.0.24",
-        "org.apache.spark:spark-sql-kafka-0-10_2.12:3.3.0",
-        "io.delta:delta-core_2.12:2.1.0",
-        "com.johnsnowlabs.nlp:spark-nlp_2.12:4.2.2",
-        "org.apache.spark:spark-hadoop-cloud_2.12:3.3.1",
-        "com.amazonaws:aws-java-sdk-bundle:1.12.339",
-        "com.databricks:spark-xml_2.12:0.15.0",
-    ]
+    # These jar files are already contained in the imranq2/helix.spark image
+    # jars = [
+    #     "mysql:mysql-connector-java:8.0.24",
+    #     "org.apache.spark:spark-sql-kafka-0-10_2.12:3.3.0",
+    #     "com.johnsnowlabs.nlp:spark-nlp_2.12:4.2.2",
```

Comment:
why did we remove delta.io?

Reply:
helix.spark base image already has delta-spark installed. Some jars versions didn't match with what's present in the base image. Therefore removed/commented unnecessary jars as they were taking a long time to get downloaded everytime the tests ran.

Reply:
@imranq2 Please let me know if we're good here and then I'll merge.